telegram/bot.py

```diff
@@ -1985,14 +2008,24 @@ def answer_pre_checkout_query(self, pre_checkout_query_id, ok, error_message=Non
             :class:`telegram.TelegramError`
 
         """
-        url = '{0]/answerPreCheckoutQuery'.format(self.base_url)
+
+        if ((ok is not True and error_message is None) or
+                (ok is True and error_message is not None)):
+            raise TelegramError(
+                'answerPreCheckoutQuery: If ok is True, there should '
+                'not be error_message; if ok is False, error_message '
+                'should not be empty')
```

Comment:
Warning seems better... especially since we don't raise errors anywhere else if arguments are wrong, so it would be a bit strange to do it here imo